pkgs/servers/heisenbridge/default.nix

```diff
@@ -1,13 +1,13 @@
 { lib, fetchFromGitHub, fetchpatch, python3 }:
 python3.pkgs.buildPythonApplication rec {
   pname = "heisenbridge";
-  version = "1.12.0";
+  version = "1.13.0";
 
   src = fetchFromGitHub {
     owner = "hifi";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-10y0sWpOLbjbGBpOjr1/tJrhcOsCwcm2sSO5kHfmJZA=";
+    rev = "refs/tags/v${version}";
```

Comment:
Why the full path here? Is this needed?

Reply:
Maybe it was a change in [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) that caused that?

Reply:
I think this is caused by https://github.com/NixOS/nixpkgs/commit/7f5915b04801f383a3504a011601e2fb09dfedf8